### PR TITLE
Minimum posting interval

### PR DESCRIPTION
**Pull request by annando**
We can now define a minimum posting interrval for mirrored feed posts. This is valid per user.

### CODE REVIEW

File: database.sql [hunk 1]
```diff
@@ -461,6 +461,19 @@ CREATE TABLE IF NOT EXISTS `conversation` (
...
+--
+CREATE TABLE IF NOT EXISTS `delayed-post` (
+	`id` int unsigned NOT NULL auto_increment,
+	`uri` varchar(255) COMMENT 'URI of the post that will be posted later',
+	`uid` mediumint unsigned COMMENT 'Owner User id',
+	`delayed` datetime COMMENT 'delay time',
+	 PRIMARY KEY(`id`),
+	 UNIQUE INDEX `uid_uri` (`uid`,`uri`(190)),
+	FOREIGN KEY (`uid`) REFERENCES `user` (`uid`) ON UPDATE RESTRICT ON DELETE CASCADE
+) DEFAULT COLLATE utf8mb4_general_ci COMMENT='Posts that are about to be posted at a later time';
```
Comment (MrPetovan) [inline database.sql]: posted = received (inbound) or distributed (outbound)?

Comment (annando) [reply database.sql]: Outbound

**Comment by annando** [comment]
BTW: Squeet.me is running with this since yesterday. It seem to work fine. The posts are now more evenly distributed, there are nearly no "bulk postings" anymore where some account is firing a bunch of 5-10 posts at one time.

**Comment by annando** [comment]
@MrPetovan I'm currently working at another branch which doesn't allow me to switch back to this branch. So I hope you won't find anything bad in this PR to change :-)

**Review by MrPetovan — APPROVED** [review]

**Comment by AlfredSK** [comment]
> Minimum interval between two feed posts per user

@annando  Is the interval set in seconds or minutes? What is a reasonable value? Any recommendation?

**Comment by annando** [comment]
It's in minutes. I set it to 10 minutes for squeet.me.

**Comment by AlfredSK** [comment]
Thank you!

**Comment by ghost** [comment]
Where can this value be set?

**Comment by annando** [comment]
You have to set it in your `local.config.php` in the `system` section. The default value is set (like always) in `static/defaults.config.php`. Please don't change it there since this will be overwritten by updates.

**Pull request merged**